src/box/lua/console.lua

```diff
@@ -635,15 +658,26 @@ local function local_read(self)
             break
         end
         if delim == "" then
-            local lang = self.language
-            if not lang or lang == 'lua' then
-                -- stop once a complete Lua statement is entered
-                if local_check_lua(buf) then
+            local multiline_delim
+            if (self.remote ~= nil) then
```

Comment:
Nit: Please, don't use redundant outmost parentheses in an if-statement in Lua.

Reply:
Fixed.